extensions/vscode/src/api/resources/Configurations.ts

```diff
@@ -70,8 +70,8 @@ export class Configurations {
   // 400 - bad request
   // 500 - internal server error
   inspect(
+    params: { dir: string; entrypoint?: string; recursive?: boolean },
     python?: string,
-    params?: { dir?: string; entrypoint?: string; recursive?: boolean },
```

Comment:
I found this switch very confusing. The Axios functions, and all of our others do `data` or parameters to fill the data/URL, and then params. Seeing the function change elsewhere really got me scratching my head before I came back here to discover the movement.

Reply:
I did not like it either, but it comes at the language requirement to maintain the order of required then optional parameters. I'm good with refactoring this to avoid this awkwardness, but I do want to maintain the typescript parameter checks as they were invaluable to the refactoring work I was doing and I believe to what we have yet to do in the future.